cli_reference/basic_cli_operations.adoc

```diff
@@ -467,6 +467,20 @@ $ oc run NAME --image=<image> \
     [options]
 ----
 
+=== patch
+Updates one or more fields of an object using strategic merge patch:
+----
```

Comment:
add a space before the code block to ensure that the formatting renders properly